<launch>
    <remap from="image" to="main_camera/image_raw"/>
    <remap from="camera_info" to="main_camera/camera_info"/>

    <node pkg="nodelet" type="nodelet" name="aruco_pose" args="load aruco_pose/aruco_pose nodelet_manager" clear_params="true">
        <param name="frame_id" value="aruco_map_raw"/>
        <param name="type" value="gridboard"/>
        <param name="markers_x" value="1"/>
        <param name="markers_y" value="6"/>
        <param name="first_marker" value="240"/>
        <param name="markers_side" value="0.3362"/>
        <param name="markers_sep" value="0.46"/>

        <!-- Custom gridboard: -->
        <!--<rosparam param="marker_ids">[6, 5, 4, 3, 2, 1]</rosparam>-->
    </node>

    <node pkg="nodelet" type="nodelet" name="aruco_vpe" args="load clever/aruco_vpe nodelet_manager" clear_params="true">
        <param name="aruco_orientation" value="map"/>
        <!--<param name="aruco_orientation" value="map_upside_down"/>-->

        <param name="use_mocap" value="true"/>
    </node>
</launch>
